internal/proto/query_coord.proto

```diff
@@ -305,6 +305,7 @@ message LoadSegmentsRequest {
   bool need_transfer = 11;
   LoadScope load_scope = 12;
   repeated index.IndexInfo index_info_list = 13;
+  bool is_lazy_load = 14;
```

Comment:
For mmap, we use TypeParams in FieldSchema.

what about direcly add coll.Properties to CollectionSchema, this could help for further expand if we have more collection params